tests/integration/docker/user.bats

```diff
@@ -45,3 +45,22 @@ setup() {
 	[ "${status}" -eq 0 ]
 	[[ "${output}" == 'root'* ]]
 }
+
+@test "Run with additional groups" {
+	run $DOCKER_EXE run --rm --group-add audio --group-add nogroup busybox id
```

Comment:
These groups are meant to exist. "--group-add" options is for running the container with these additional groups added to the user's groups.